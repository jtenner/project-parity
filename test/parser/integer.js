const { OptionType } = require("../../lib");

module.exports = {
  argv: ["--option-a", "10"],
  config: {
    optionA: {
      type: OptionType.Integer,
      default: [],
    },
  },
};
