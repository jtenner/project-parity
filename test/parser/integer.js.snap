{
  args: [],
  cliProvided: Set(1) { 'optionA' },
  configProvided: Set(0) {},
  errors: [],
  options: { optionA: 10 },
  unknown: []
}
